### ISSUE TITLE
how to use this with my own dataset?

### ISSUE THREAD

**Issue by manvirvirk** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#35]
i want to train it with my own dataset. Can anyone help?

**Comment by bigmb** [comment]
Just add your data in the folder mentioned in pytorch_run.py

**Comment by manvirvirk** [comment]
thanks for the help. Can you please tell me the line number??

**Comment by manvirvirk** [comment]
my training data consist of only biomedical RGB images on which i want to perform segmentation.

**Comment by bigmb** [comment]
Read the description. I have written the line number.

**Comment by manvirvirk** [comment]
GOT IT, THANKS

**Comment by manvirvirk** [comment]
do we need input image labels also? my dataset contain only rgb images with labels?

**Comment by bigmb** [comment]
Sorry I couldn't get you?
you need ground truth as input too.

**Comment by manvirvirk** [comment]
my question is that not all datasets have corresponding ground truth images. What to do in that case? As in my case i have collected database from some clinic .

**Comment by bigmb** [comment]
Without GT you can do supervised training. You need to go for unsupervised training.
